components/esp_rom/include/esp32c3/rom/secure_boot.h

```diff
@@ -25,7 +25,7 @@ extern "C" {
 #endif
 
 struct ets_secure_boot_sig_block;
-struct ets_secure_boot_signature_t;
+struct ets_secure_boot_signature;
```

Comment:
I would suggest that we remove both these lines. Followup `typedef` should take care of appropriate forward declaration. Can you please check once and update this PR accordingly? @lovyan03

Reply:
@mahavirj 
thanks !
I have tested your suggestion.
You are indeed right, removing those two lines solves the error.

By the way, this has nothing to do with this directly, Only the esp32s3 file has the include guard set to " pragma once ".
For other processors, it is " ifndef _ROM_SECURE_BOOT_H_ ".
Is there any intention to do this?
If you don't mind, I'd like to unify this in this pull request as well.

Reply:
@lovyan03 

Makes sense! Please update for `pragma once` in all headers. Thanks.

Reply:
@mahavirj 
thanks!
I have completed the fix. Please confirm.